server/config/redis.js

```diff
@@ -0,0 +1,5 @@
+const Redis = require("ioredis");
+
+const redisClient = new Redis();
```

Comment:
Write complete code to configuration redis through cloud it will only works for localhost i.e. Redis should be running on machine if we use cloud one it will not work

Reply:
Ok then fine, leave it.